h2/src/main/org/h2/mvstore/tx/VersionedValueType.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2004-2017 H2 Group. Multiple-Licensed under the MPL 2.0,
```

Comment:
You missed one.

Not a big problem, of course, but it's better to have consistent headers to simplify search and replace after new year.